lib/SPIRV/SPIRVReader.cpp

```diff
@@ -685,15 +685,15 @@ void SPIRVToLLVM::setLLVMLoopMetadata(const LoopInstType *LM,
     assert(NumParam <= LoopControlParameters.size() &&
            "Missing loop control parameter!");
   }
-  if (LC & InitiationIntervalINTEL) {
+  if (LC & LoopControlInitiationIntervalINTEL) {
```

Comment:
They also present in SPIRVWriter.cpp. And in in comments of some tests (which is not a problem but may be fixed as well to be aligned with enum labels).